addons/pos_restaurant/static/tests/tours/pos_restaurant_tour.js

```diff
@@ -161,15 +161,10 @@ registry.category("web_tour.tours").add("pos_restaurant_sync", {
             Chrome.clickMenuOption("Orders"),
             // The order ref ends with -0002 because it is actually the 2nd order made in the session.
             // The first order made in the session is a floating order.
-            TicketScreen.deleteOrder("-0002"),
+            TicketScreen.deleteOrder("Main Floor/2"),
             Dialog.confirm(),
-            {
-                ...Dialog.confirm(),
-                content:
-                    "acknowledge printing error ( because we don't have printer in the test. )",
-            },
```

Comment:
Orderlines on table 2 aren't sent to the preparation tools, so when deleting there is no popup.